src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java

```diff
@@ -24,7 +24,7 @@ public static void setConfigurationBuilder() throws CheckstyleException, IOExcep
     @Test
     public void emptyLineSeparatorTest() throws IOException, Exception {
         
-        java.lang.Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
+        Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
```

Comment:
I am surprised too, can not do investigation from phone, willtry to take a close look later